src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -537,6 +552,24 @@ protected function initializeContainer()
         }
     }
 
+    protected function cleanBotLogs(array $records = array())
```

Comment:
no need for this :) the less we do the better - this means adding all args explicitly to the error handler, ignoring $context (which is deprecated in php7.2 anyway)